src/util/graph-templates/templateI.js

```diff
@@ -6,46 +6,132 @@
  * options object.
  */
 
-import Base_Template from './baseTemplate';
+import BaseTemplate from './baseTemplate';
 import { color } from './colors';
 import d3 from '@/assets/d3';
 import util from './template-utils';
 
 // Coordinates for the three corners of the triangle.
 // First corner duplicated to complete the path
-const triangleData = [{ x: -1, y: 0 }, { x: 1, y: 0 }, { x: 0, y: 1 }, { x: -1, y: 0 }];
+const triangleData = [
+  { x: -1, y: 0 },
+  { x: 1, y: 0 },
+  { x: 0, y: 1 },
+  { x: -1, y: 0 },
+];
 
 // Coordinates for the grid lines (starting and ending positions)
 const gridLinesData = [
-  [{ x: -0.9, y: 0.1 }, { x: -0.8, y: 0 }],
-  [{ x: -0.8, y: 0.2 }, { x: -0.6, y: 0 }],
-  [{ x: -0.7, y: 0.3 }, { x: -0.4, y: 0 }],
-  [{ x: -0.6, y: 0.4 }, { x: -0.2, y: 0 }],
-  [{ x: -0.5, y: 0.5 }, { x: 0.0, y: 0 }],
-  [{ x: -0.4, y: 0.6 }, { x: 0.2, y: 0 }],
-  [{ x: -0.3, y: 0.7 }, { x: 0.4, y: 0 }],
-  [{ x: -0.2, y: 0.8 }, { x: 0.6, y: 0 }],
-  [{ x: -0.1, y: 0.9 }, { x: 0.8, y: 0 }],
-
-  [{ x: 0.1, y: 0.9 }, { x: -0.8, y: 0 }],
-  [{ x: 0.2, y: 0.8 }, { x: -0.6, y: 0 }],
-  [{ x: 0.3, y: 0.7 }, { x: -0.4, y: 0 }],
-  [{ x: 0.4, y: 0.6 }, { x: -0.2, y: 0 }],
-  [{ x: 0.5, y: 0.5 }, { x: 0.0, y: 0 }],
-  [{ x: 0.6, y: 0.4 }, { x: 0.2, y: 0 }],
-  [{ x: 0.7, y: 0.3 }, { x: 0.4, y: 0 }],
-  [{ x: 0.8, y: 0.2 }, { x: 0.6, y: 0 }],
-  [{ x: 0.9, y: 0.1 }, { x: 0.8, y: 0 }],
-
-  [{ x: -0.9, y: 0.1 }, { x: 0.9, y: 0.1 }],
-  [{ x: -0.8, y: 0.2 }, { x: 0.8, y: 0.2 }],
-  [{ x: -0.7, y: 0.3 }, { x: 0.7, y: 0.3 }],
-  [{ x: -0.6, y: 0.4 }, { x: 0.6, y: 0.4 }],
-  [{ x: -0.5, y: 0.5 }, { x: 0.5, y: 0.5 }],
-  [{ x: -0.4, y: 0.6 }, { x: 0.4, y: 0.6 }],
-  [{ x: -0.3, y: 0.7 }, { x: 0.3, y: 0.7 }],
-  [{ x: -0.2, y: 0.8 }, { x: 0.2, y: 0.8 }],
-  [{ x: -0.1, y: 0.9 }, { x: 0.1, y: 0.9 }],
```

Comment:
Can we put `/* eslint-disable */` for this block?

Reply:
oh shit, yes, most deff. I'll fix